gtdynamics/utils/ChainInitializer.cpp

```diff
@@ -27,30 +27,35 @@ gtsam::Values ChainInitializer::ZeroValues(
 
   // Initialize link dynamics to 0.
   for (auto&& link : robot.links()) {
-    int i = link->id();
-    InsertPose(&values, i, t, AddGaussianNoiseToPose(link->bMcom(), sampler));
-    InsertTwist(&values, i, t, sampler.sample());
-    InsertTwistAccel(&values, i, t, sampler.sample());
+    const int i = link->id();
+    if (i==0 || i==3 || i==6 || i==9 || i==12)
+      InsertPose(&values, i, t, AddGaussianNoiseToPose(link->bMcom(), sampler));
+    //InsertTwist(&values, i, t, sampler.sample());
```

Comment:
Since you've moved these lines outside the if statement, kill these commented out lines?